docker/config/nginx/nginx.conf

```diff
@@ -25,7 +25,12 @@ http {
 
     keepalive_timeout  65;
 
-    #gzip  on;
+    gzip on;
+    gzip_disable "MSIE [1-6]\\.(?!.*SV1)";
```

Comment:
Qual a razão de desabilitar o gzip aqui?

Reply:
Não encontrei o motive dele e de outras configurações (https://www.techrepublic.com/article/how-to-configure-gzip-compression-with-nginx/, https://github.com/defunkt/unicorn/blob/master/examples/nginx.conf e https://wordpress.org/support/topic/gzip-not-enabled-2/) desabilitarem o Internet Explore.

Reply:
Vc saberia porque @LeandroJatai ?